### ISSUE TITLE
Explore Artefact Detection Tools

### ISSUE THREAD

**Issue by DanielHieber** [hnu-digihealth/patho-pix#1]
Select a tool for artefact detection and get it running.

Maybe take a look at:
- https://www.nature.com/articles/s41598-022-08351-5
- https://github.com/choosehappy/HistoQC

**Comment by joa24jm** [comment]
No. 2 referenziert auf https://github.com/MaryamHaghighat/PathProfiler, aber auch hier finde ich kein Model o. ä.
